src/views/Lottery/components/Hero.tsx

```diff
@@ -1,104 +1,279 @@
 import React from 'react'
-import styled from 'styled-components'
-import { Heading, Text, LinkExternal } from '@pancakeswap/uikit'
+import styled, { keyframes } from 'styled-components'
```

Comment:
This is a straight delete of v1 and replace with v2, but the file name is the same, so Git is parsing it as a diff

`views/Lottery/components/Hero` > deleted
`views/LotteryV2/components/Hero` >  renamed `views/Lottery/components/Hero`